bin/p2-run-hooks/main.go

```diff
@@ -19,6 +19,7 @@ var (
 	hookDir      = kingpin.Flag("hook-dir", "The root of the hooks").Default(hooks.DEFAULT_PATH).String()
 	manifestPath = kingpin.Flag("manifest", "The manifest to use (this is useful when we are in the before_install phase)").ExistingFile()
 	nodeName     = kingpin.Flag("node-name", "The name of this node (default: hostname)").String()
+	podRoot      = kingpin.Flag("pod-root", "The system root for pods (default: /data/pods").Default(pods.DefaultPath).String()
```

Comment:
nit: When you use the `.Default()` option, its value is printed in the help message. You don't need to add the extra `"(default: ...)"` message.